nearcore/src/lib.rs

```diff
@@ -74,7 +73,7 @@ fn create_db_checkpoint(path: &Path, near_config: &NearConfig) -> anyhow::Result
                     checkpoint_path.display(),
                     path.display());
 
-    let db = RocksDB::open(path, &near_config.config.store, Mode::ReadWrite)?;
+    let db = near_store::db::RocksDB::open(path, &near_config.config.store, Mode::ReadWrite)?;
```

Comment:
Is there a reason for inlining the import here and not importing RocksDB on top of the file?